gtests/net/mptcp/syscalls/sockopt_SENT_LOWAT.pkt

```diff
@@ -0,0 +1,58 @@
+// connect() function, connection initiated by the kernel
```

Comment:
about the location, should we not move it to the `sockopts` dir, instead of the `syscalls` one?